count/Feature/TopicsFeature.swift

```diff
@@ -47,26 +73,34 @@ struct TopicsFeature: Reducer {
     var body: some ReducerOf<Self> {
         Reduce { state, action in
             switch action {
-            case .about:
-                return .none
-
             case .aboutButtonTapped:
-                state.about = .init()
+                state.destination = nil
+                return .run { send in
+                    await send(.setDestination(.about(.init())))
```

Comment:
Doing this requires `Destination.State` and all its downstream types to be `@Sendable`. Way downstream the TCA built in `AlertState` type was not `@Sendable` in TCA v1.0.0, but it is in v1.3.0, so I updated TCA.